src/plugins/auth/OAuth/hook.ts

```diff
@@ -0,0 +1,15 @@
+import { auth } from './auth'
+
+type AuthState = import('./types').AuthPluginState
+type RequestOptions = import('./types').RequestOptions
```

Comment:
This is not really exported from the `./types` file. I'm facing build error:

```sh
Stack Trace:
Error: /Users/shaikat/Dev/github/MunifTanjim/node-bitbucket/src/plugins/auth/OAuth/hook.ts(4,41): semantic error TS2694: Namespace '"/Users/shaikat/Dev/github/MunifTanjim/node-bitbucket/src/plugins/auth/OAuth/types"' has no exported member 'RequestOptions'.
    at error (/Users/shaikat/Dev/github/MunifTanjim/node-bitbucket/node_modules/rollup/dist/rollup.js:9435:30)
    at throwPluginError (/Users/shaikat/Dev/github/MunifTanjim/node-bitbucket/node_modules/rollup/dist/rollup.js:15754:12)
```

Reply:
Will be fixed in the next commit